### DISCUSSION TITLE
MTF Failures on Large HHs due to CDAP+WFH Model

### DISCUSSION THREAD

**Discussion by AndrewTheTM** [synthicity/activitysim#619] [General]
Apologies for the alphabet soup in the subject line...

I've run into an issue in MWCOG and Metropolitan Council where the mandatory tour frequency model fails because of people working from home in large households. The extended probability table (cdap_fixed_relative_proportions.csv) is usually setup with a probability of mandatory, non-mandatory, and home by person type. This table is *usually* agnostic of work from home, which is the source of the problem - a 6+ person HH may set a worker to have a mandatory CDAP pattern but no work location so MTF fails because normally the utility of work1 and work2 are -999 for workers with no work location. 

The attached file has additional lines for full-time and part-time workers that do/do not work from home (I also have a line for preschool children, which I ran into a similar issue with Met Council with preschool children).  

[cdap_fixed_relative_proportions.csv](https://github.com/ActivitySim/activitysim/files/9758332/cdap_fixed_relative_proportions.csv)

**Comment by jpn--** [comment]
I guess the real question is whether a worker in a household of 6 or more would ever work from home?  😜 

Want to open a PR with this change?  It's just a drop-in replacement of the named file in the relevant models, right?  Given the probably limited number of workers who are person 6+ in their household and who WFH, it likely has limited impact on model calibration or other results.

**Reply by AndrewTheTM** [reply]
Regarding the first question, I can tell you from experience that it's not about the number of people, it's about the number and type of people *at home*! It is more difficult to work from home when you have three kids running around the floor above you then when it's an adult/young adult reading a book or watching TV.

I'll update this issue a pull request in a day or two - there's a few other things that will be in that PR for MWCOG as well that I'm preparing.
